Auth0.podspec

```diff
@@ -69,18 +68,18 @@ Pod::Spec.new do |s|
   s.requires_arc = true
 
   s.ios.source_files = 'Auth0/*.{swift,h,m}', 'Auth0/ObjectiveC/*.{h,m}'
-  s.ios.frameworks = 'UIKit', 'SafariServices', 'LocalAuthentication'
-  s.ios.weak_framework = 'AuthenticationServices'
```

Comment:
Now that all platforms that support Web Auth use only `ASWebAuthenticationSession` (AuthenticationServices), it can go in `frameworks` instead of `weak_frameworks`.